src/crimson/osd/heartbeat.cc

```diff
@@ -309,6 +312,29 @@ seastar::future<> Heartbeat::handle_you_died()
   return seastar::now();
 }
 
+void Heartbeat::heartbeat_check()
+{
+  failure_queue_t failure_queue;
+  auto now = clock::now()
```

Comment:
this does not compile.
```suggestion
  const auto now = clock::now();
```